### PR TITLE
Add info fields to ST-owned drivers

### PR DESCRIPTION
**Pull request by greens**

### CODE REVIEW

**Comment by greens** [comment]
image

This is roughly what the updated fields will look like. It looks like the plugin does some automatic capitalization.

File: drivers/SmartThings/matter-lock/config.yml [hunk 1]
```diff
@@ -2,3 +2,5 @@ name: 'Matter Lock'
 packageKey: 'matter-lock'
 permissions:
   matter: {}
+description: description: "Smartthings driver for Matter lock devices"
```
Comment (cjswedes) [inline drivers/SmartThings/matter-lock/config.yml:5]: Why is `description: ` in there twice?

File: drivers/SmartThings/bose/config.yml [hunk 2]
```diff
@@ -3,3 +3,5 @@ packageKey: 'bose'
 permissions:
   lan: {}
   discovery: {}
+description: "Smartthings driver for Bose devices"
```
Comment (tpmanley) [inline drivers/SmartThings/bose/config.yml:6]:
```suggestion
description: "SmartThings driver for Bose devices"
```
Here and everywhere else

File: drivers/SmartThings/zwave-bulb/config.yml [hunk 3]
```diff
@@ -2,3 +2,5 @@ name: 'Z-Wave Bulb'
 packageKey: 'zwave-bulb'
 permissions:
   zwave: {}
+description: "Smartthings driver for Zigbee lightbulb devices"
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-bulb/config.yml:5]:
```suggestion
description: "SmartThings driver for Z-Wave light bulb devices"
```

File: drivers/SmartThings/zwave-button/config.yml [hunk 4]
```diff
@@ -2,3 +2,5 @@ name: 'Z-Wave Button'
 packageKey: 'zwave-button'
 permissions:
   zwave: {}
+description: "Smartthings driver for Zigbee button and remote devices"
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-button/config.yml:5]:
```suggestion
description: "SmartThings driver for Z-Wave button and remote devices"
```

File: drivers/SmartThings/zwave-garage-door-opener/config.yml [hunk 5]
```diff
@@ -2,3 +2,5 @@ name: 'Z-Wave Garage Door Opener'
 packageKey: 'zwave-garage-door-opener'
 permissions:
   zwave: {}
+description: "Smartthings driver for Z-Wave fan devices"
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-garage-door-opener/config.yml:5]:
```suggestion
description: "SmartThings driver for Z-Wave garage door opener devices"
```

File: drivers/SmartThings/zwave-bulb/config.yml [hunk 6]
```diff
@@ -2,5 +2,5 @@ name: 'Z-Wave Bulb'
 packageKey: 'zwave-bulb'
 permissions:
   zwave: {}
-description: "Smartthings driver for Zigbee lightbulb devices"
+description: "SmartThings driver for Zigbee lightbulb devices"
```
Comment (tpmanley) [inline drivers/SmartThings/zwave-bulb/config.yml:5]:
```suggestion
description: "SmartThings driver for Z-Wave lightbulb devices"
```

**Review by tpmanley — APPROVED** [review]

**Review by ctowns — APPROVED** [review]

**Pull request merged**